run/mainnet/nix/.env

```diff
@@ -1 +1,3 @@
 NETWORK="mainnet"
+AGGREGATOR_ENDPOINT=https://aggregator.release-mainnet.api.mithril.network/aggregator
```

Comment:
this is the same env variables like for docker - we don't want to reuse them?

Reply:
reusing the .env is possible with symlinks, but I'm not sold on the design of this `run` directory yet. I will happily reconsider this in future PRs. Good point anyway.